RetailCoder.VBE/Common/LogLevelHelper.cs

```diff
@@ -45,15 +45,21 @@ public static int MaxLogLevel()
 
         public static void SetDebugInfo(String value)
         {
-            DebugInfo = value;
+            LogHeader = value;
             LogHeaderWritten = false;
         }
 
         public static void SetMinimumLogLevel(LogLevel minimumLogLevel)
         {
+            if (LogManager.GlobalThreshold == minimumLogLevel) 
```

Comment:
Per the NLog docs: GlobalThreshold: Gets or sets the global log threshold. Log events below this threshold are not logged.